src/theme/MuiThemeProvider/muiTheme.ts

```diff
@@ -121,6 +121,7 @@ export default createTheme({
     h4: {
       fontSize: '1.25rem',
       fontWeight: 600,
+      lineHeight: '1.875rem',
```

Comment:
That corresponds to the line-height that should already be applied by default to all elements (1.5).

Reply:
Yeah, that's 1.5 its font size (20)